src/style.css

```diff
@@ -0,0 +1,125 @@
+body {
+  background-color: #f6f6f6;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  color: #444;
+  height: 100vh;
+  font-family: 'Times New Roman', Times, serif;
+}
+
+input {
+  font-style: italic;
+  border: none;
+  padding: 1em;
+  font-size: 1em;
+  outline: none;
+}
+
+ul {
+  padding: 0;
+  margin: 0;
+}
+
+li {
+  padding: 1em;
+  margin: 0;
+  list-style: none;
+}
+
+.title,
+input,
+li {
+  border-bottom: lightgray solid 1px;
+}
+
+label {
+  margin-left: 0.5em;
+}
+
+.title {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  padding: 1em 1em;
+  margin: 0;
+  box-sizing: border-box;
+  width: 100%;
+}
+
+.title > p {
+  margin: 0;
+}
+
+.itemElement {
+  display: flex;
+  justify-content: space-between;
+}
+
+.iconContainer {
+  display: flex;
+  align-items: center;
+  padding: 0;
+  width: min-content;
+  cursor: pointer;
+}
+
+.checkboxContainer {
+  display: flex;
+}
+
+.all {
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+  align-items: flex-start;
+  width: 25rem;
+  background-color: #fff;
+  box-shadow: 0 5px 20px -5px rgba(0, 0, 0, 0.76);
+  -webkit-box-shadow: 0 5px 20px -5px rgba(0, 0, 0, 0.76);
+  -moz-box-shadow: 0 5px 20px -5px rgba(0, 0, 0, 0.76);
+}
+
+.containerTodo {
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+}
+
+.btnCom {
+  border: none;
+  background-color: #fff;
+  cursor: pointer;
+  align-self: center;
+  margin: 1em;
+}
+
+.textInput {
+  font-family: 'Times New Roman', Times, serif;
+  color: #444;
```

Comment:
- [ ] Following the guideline on following HTML and CSS best practices and DRY best practices, I believe that you only need to declare your font-family and color once. The reason for this is that one copy is easier to maintain than multiple copies; if the information need to be changed, there is only one place to change it. Kindly look into it and fix.
-The font family and color was first declared here.
https://github.com/eric2umeh/to-do-list/blob/36d75b39d93da6abcd5eab47a07cb45345b00e36/src/style.css#L1-L10 

- It was declared here for the second time. 
https://github.com/eric2umeh/to-do-list/blob/36d75b39d93da6abcd5eab47a07cb45345b00e36/src/style.css#L99-L102

Reply:
Thank you